src/main/kotlin/com/msg/gcms/domain/webhook/util/impl/DiscordUtilImpl.kt

```diff
@@ -0,0 +1,61 @@
+package com.msg.gcms.domain.webhook.util.impl
+
+import com.msg.gcms.domain.club.enums.ClubType
+import com.msg.gcms.domain.webhook.config.DiscordWebhookConfig
+import com.msg.gcms.domain.webhook.util.DiscordUtil
+import okhttp3.MediaType.Companion.toMediaType
+import okhttp3.OkHttpClient
+import okhttp3.Request
+import okhttp3.RequestBody.Companion.toRequestBody
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.stereotype.Component
+
+@Component
+class DiscordUtilImpl(val httpClient: OkHttpClient): DiscordUtil{
+
```

Comment:
```suggestion
@Component
class DiscordUtilImpl(
    private val httpClient: OkHttpClient
): DiscordUtil{

```
로 접근 제한둬서 하는거 어떤가요? 물론 DI로 이뤄질거라 접근될지에 대해서는 걱정은 안되긴하는데.. private가 안심되죠

Reply:
그러네요... 막연하게 괜찮겠지라고 생각한😅 바꾸도록 할게요!